chrome/locale/en-US/zotero/zotero.ftl

```diff
@@ -436,4 +436,7 @@ toggle-preview =
 
 quickformat-aria-bubble = Press Down Arrow to open citation properties.
 quickformat-aria-input = Type your search term. Use Down Arrow and Up Arrow to navigate the search results.
-quickformat-aria-item = Press { return-or-enter } to select this item. Press Tab to return to the search field.
+quickformat-aria-item = { $section ->
+    [none] { "" }
+    *[other] This item belongs to { $section } section. Press { return-or-enter } to select this item. Press Tab to return to the search field.
```

Comment:
I haven't been able to try this in VoiceOver yet, but do you have to press something to get this announced? Is `This item belongs to { $section } section.` a common sort of announcement? If so, how about just `This item belongs to { $section }.`? E.g., "This item belongs to Selected Items." or "This item belongs to My Library.".